src/js/components/Fragments/Navbuttons.scss

```diff
@@ -1,34 +1,31 @@
-.nav-buttons{
+.Navbuttons {
     position: absolute;
     bottom: 20px;
     width: 100%;
     overflow: hidden;
     max-height: 60px;
-    background-color: transparent;
-    text-align:center;
+    background-color: #eee0;
+    text-align: center;
 }
-
-.btn-nav-button{
+.btnNavButton {
```

Comment:
Why did you remove the spacing between declarations? please add it back

Reply:
sorry was with merge conflict, fixed.